packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -310,6 +310,13 @@ const PostsPage = ({post, refetch, classes}: {
   const welcomeBoxProps = welcomeBoxABTestGroup === "welcomeBox" && !currentUser && maybeWelcomeBoxProps;
   const welcomeBox = welcomeBoxProps ? <WelcomeBox {...welcomeBoxProps} /> : null;
 
+  // If this is a non-AF post being viewed on AF, redirect to LW.
+  const isAF = (forumTypeSetting.get() === 'AlignmentForum');
+  if (isAF && !post.af) {
+    const lwURL = "https://www.lesswrong.com" + location.url;
```

Comment:
I assume this works during SSR, but double-checking

Reply:
Yep, it renders as an http 302 redirect.